terraform/install.sh

```diff
@@ -174,6 +174,24 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  external_ip="";
```

Comment:
you should display a message to the user so they know what they're waiting on

Reply:
Added logging message :)